src/libs/CardUtils.ts

```diff
@@ -29,7 +29,7 @@ function getMonthFromExpirationDateString(expirationDateString: string) {
  * @param cardID
  * @returns boolean
  */
-function isExpensifyCard(cardID: string) {
+function isExpensifyCard(cardID: number) {
```

Comment:
I noticed that the `getCardDescription` function expects a cardID of type string. Should we also check and adjust the typing on line 44 to number?